gpu4pyscf/tdscf/ris.py

```diff
@@ -747,18 +775,79 @@ def __init__(self,
         self.nstates = nstates
         self.max_iter = max_iter
         self.mol = mf.mol
+        self.mo_coeff = cp.asarray(mf.mo_coeff, dtype=self.dtype)
         self.spectra = spectra
         self.out_name = out_name
         self.print_threshold = print_threshold
-        self.GS = GS
+        self.gram_schmidt = gram_schmidt
         self.group_size = group_size
         self.group_size_aux = group_size_aux
 
-        self.verbose = mf.verbose
+        self.verbose = verbose if verbose else mf.verbose
+
         self.device = mf.device
+        
+        self._in_ram = in_ram
```

Comment:
The value of this parameter and some of the parameters (spectra, out_name, etc.) are not printed anywhere. They can be logged in the build() function. Some attributes like n_occ, n_vir should also be dumped into the log at INFO level